sqlserver/tests/odbc/odbcinst.ini

```diff
@@ -2,4 +2,4 @@
 FreeTDS=Installed
 
 [FreeTDS]
-Driver=/usr/lib/x86_64-linux-gnu/odbc/libtdsodbc.so
+Driver=/usr/lib64/libtdsodbc.so
```

Comment:
Is there a story to this change?

Reply:
oh this is my bad, let me fix it :hat-tip: